dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/frameworks/Lksg.kt

```diff
@@ -113,7 +113,7 @@ class Lksg {
 
     private fun sortComplaintRisksInDataset(dataset: LksgData): LksgData {
         val complaintRisksIdentifiedRisks = dataset.governance?.grievanceMechanismOwnOperations
-            ?.complaintsRiskPosition!!.toMutableList()
+            ?.complaintsRiskPosition?.toMutableList() ?: mutableListOf()
```

Comment:
why this change? i though this would be the same in terms of functionality

Reply:
No. The previous one crashes when complaintsRiskPosition is null because of the non-null-assertion